src/baseline_bots/bots/smart_order_accepter_bot.py

```diff
@@ -287,6 +290,102 @@ def generate_support_proposals(self, comms_obj: MessagesData) -> Dict[str, str]:
                 final_messages[recipient] = str(suggested_proposals)
                 comms_obj.add_message(recipient, str(suggested_proposals))
         return final_messages
+        
+    def is_order_aggressive_to_powers(self, order: str, powers: List[str]):
+        """
+        check if the order is aggressive by
+        1. to attack allies unit
```

Comment:
Is this saying "if the order is attacking an ally's unit"?

Reply:
yeah that's more correct! I am gonna change that